tests/e2e/lib.sh

```diff
@@ -115,13 +115,22 @@ deploy_stackrox_operator() {
     if [[ "${DEPLOY_STACKROX_VIA_OPERATOR}" == "false" ]]; then
         return
     fi
+    
+    if [[ "${USE_MIDSTREAM_IMAGES}" == "true" ]]; then
+        info "Deploying ACS operator via midstream image"
+        # hardcoding values for testing
+        export VERSION="545338"
```

Comment:
@robbycochran How do you suggest we pass the appropriate `iib` version number? I guess it would be ideal if it is exported from the `build` cluster.

Reply:
Hi @mdafsanhossain @robbycochran @jdao-rh 

IIB tag is the way we reference a specific mid/downstream build of ACS. IIB tags are opaque, i.e. I don't think we can predict the number from the build.

As such, I think it should be fine to have the tag hardcoded _somewhere_ and update it any time we need to use more recent builds. Obviously, updating the tag would require a PR but if it's just tag change, it should be quick to approve.

A caveat is that IIB tags are different per OCP version, therefore, it should be not just one tag but a map `ocp version -> iib tag`. Then, the correct tag should be looked up according to the OCP version tests would run on. Otherwise, we risk trying to apply index to a different ocp version breaking something.

An alternative to hardcoding IIB tag would be to build an index image ad-hoc with given rhacs operator bundle image. This way we won't depend on arbitrary IIB tags but likely will have only ACS operator in that catalog.
Index images are built with `opm` command. The example how to do it is available here
https://github.com/stackrox/stackrox/blob/b691ad596c42583cc80f24958fe6a642c258fac8/operator/Makefile#L431-L447
See the related [build-index-image.sh](https://github.com/stackrox/stackrox/blob/b691ad596c42583cc80f24958fe6a642c258fac8/operator/hack/build-index-image.sh) command.

Just wanted to share my thoughts since it was mentioned in today's tech interlock meeting. I suggest ultimately the idea how to reference mid/downstream build in upstream tests should fit in the broader vision. Hoping for @robbycochran to provide input here

Reply:
@msugakov I was thinking in a similar way to your first approach. I am thinking of creating a JSON containing the appropriate values. I will check out your second approach as well.